### ISSUE TITLE
Does the custom web receiver support Fairplay DRM for the HLS stream?

### ISSUE THREAD

**Issue by lalit-tudip** [googlecast/CastReceiver#56]
I created a custom web receiver that supports Widevine DRM for the DASH stream(cast from the Android mobile). I need to add support for iOS mobile for HLS streams with Fairplay DRM, Is it possible as there is no documentation available for it?

**Comment by SimonAllen0901** [comment]
No, I don't think the custom web receiver supports Fairplay DRM for HLS streams 😞

Chromecast's Web Receiver SDK primarily supports Widevine DRM for DASH streams. Fairplay DRM is exclusive to Apple platforms (e.g., Safari on iOS/macOS) and is not supported on Chromecast receivers, which are based on Chromium.
